docs/V3/reference/error-codes.md

```diff
@@ -0,0 +1,51 @@
+---
+id: error-codes
+title: Error Codes
+---
+
+
+LiquidityMath.sol
+
+* `LS` : Liquidity Sub
+* `LA`: Liquidity Add
+
+Oracle.sol
+
+* `OLD`: The target must be chronologically at or after the oldest observation
+
+* `I`: Cardinality must be greater than 0
+  
+Position.sol
+
+* `NP`: disallow pokes for 0 liquidity positions
+
+Tick.sol
+
+* `LO` LiquidityGrossAfter must be less than MaxLiquidity
+
+TickMath.sol 
+
+* `T`:  require(absTick <= uint256(MAX_TICK), 'T');
+* `R`: second inequality must be < because the price can never reach the price at the max tick
```

Comment:
needs info about why we have a limited tick range, this comment is not an explanation of why do the require